container/lxd/network_test.go

```diff
@@ -172,9 +173,9 @@ func (s *networkSuite) TestVerifyNetworkDeviceIPv6Present(c *gc.C) {
 
 	err = jujuSvr.VerifyNetworkDevice(defaultProfile(), "")
 	c.Assert(err, gc.ErrorMatches,
-		`profile "default": no network device found with nictype "bridged" or "macvlan", `+
-			`and without IPv6 configured.\n`+
-			`\tthe following devices were checked: \[eth0\]`)
+		`profile "default": juju does not support IPv6. Disable IPv6 in LXD via:\n`+
+			`\tlxc network set lxdbr0 ipv6.address none\n`+
+			`and run the command again`)
```

Comment:
vs "the command" do we know what was run? Is this always going to be bootstrap? Can this be "run bootstrap again"?

Reply:
This message can also occur from tools/lxdclient.Connect()... used when we create an lxd container on a juju machine.  Given that we create those lxd profiles, very unlikely to be hit.  Not sure what command would be run again if that's hit.